chsdi/templates/htmlpopup/gebaeuderegister.mako

```diff
@@ -4,23 +4,28 @@
 
 <%def name="table_body(c, lang)">
 <%
-c['stable_id'] = True
-street_key = 'strname1'
-if 'strname_de' in c['attributes']:
-  lang = lang if lang not in ['fr', 'it', 'rm'] else 'de'
-  if c['attributes']['strname_%s' % lang]:
-    street_key = 'strname_%s' % lang
+  c['stable_id'] = True
+  street_key = 'strname1'
+  lang = lang if lang in ('fr', 'it', 'rm') else 'de'
+  topic = request.matchdict.get('map')
+  canton = c['attributes']['gdekt']
+  bfs_nr = c['attributes']['gdenr']
+  url_canton = 'https://data.geo.admin.ch/ch.bfs.gebaeude_wohnungs_register/CSV/%s/%s.zip' % (str(canton), str(canton))
+  url_municipality = 'https://data.geo.admin.ch/ch.bfs.gebaeude_wohnungs_register/CSV/%s/%s.zip' % (str(canton), str(bfs_nr))
```

Comment:
you can access data geoadmin host via `request.re
gistry.settings['datageoadminhost']`

Reply:
you don't need `str(canton)`, this is already the case when using `%s`

Reply:
Thx for your feedback! It's done. I just need to deploy the branch on int to update the testlink.
It was still in WIP because I was expecting a feedback before the deploy ;-)